cl/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "courtlistener-webpack-loader",
+  "version": "1.0.0",
+  "description": "use modern js frameworks with courtlistener",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "tests"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "scripts": {
+    "build": "NODE_ENV=production ./node_modules/.bin/webpack --config webpack.config.js",
+    "dev": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --hot",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.11.4",
+    "@babel/plugin-transform-runtime": "^7.11.0",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/preset-typescript": "^7.10.4",
+    "@babel/runtime": "^7.11.2",
+    "@types/js-cookie": "^2.2.6",
+    "@types/node": "^14.6.0",
+    "@types/react": "^16.9.46",
+    "@types/react-dom": "^16.9.12",
+    "@types/react-router-dom": "^5.1.5",
+    "@typescript-eslint/eslint-plugin": "^3.9.1",
+    "@typescript-eslint/parser": "^3.9.1",
+    "babel-loader": "^8.1.0",
+    "clean-webpack-plugin": "*",
+    "compression-webpack-plugin": "^5.0.1",
+    "downshift": "^6.0.5",
+    "eslint": "^7.7.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.20.6",
+    "js-cookie": "^2.2.1",
+    "prettier": "^2.0.5",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-query": "^2.8.0",
+    "react-router-dom": "^5.2.0",
+    "react-virtual": "^2.2.1",
+    "typescript": "^4.2.4",
+    "webpack": "^4.44.1",
+    "webpack-bundle-analyzer": "^4.4.1",
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.11.0"
+  },
+  "dependencies": {
```

Comment:
Can you go through and make sure that all of these dependencies are in the right place? For example, do we need types in our dependencies or in our dev deps? My assumption is we can lighten the JS we ship by checking this a little more?

Reply:
So it shouldn't matter where the packages are placed. The way webpack works is that it will (or should) only bundle imported modules (often referred to as "tree-shaking"). Meaning that even if a package is in "Dependencies" and not dev dependencies, the only affect it will have will be on the server at build time (i.e., deployment). Once the JS is compiled and minified, nothing else is added into it.

You can see what the bundle sizes are by running the webpack-bundle-analyzer.

Reply:
@mlissner what @litewarp said is what I also see.  Although I did try to purge anything that we weren't using, I did not touch anything like Webpack-cli

Reply:
Makes sense regarding bundling. I didn't mean to just highlight the webpack stuff though, Bill. Could you just eye the list of all packages and see if there are ones to remove?

Reply:
absolutely

Reply:
two dependencies look superfluous to me - and to a dep check `i` and `caniuse`